app/components/CoachList.tsx

```diff
@@ -27,6 +28,7 @@ export default function CoachList({ coaches }: { coaches: ICoach[] }) {
               emergency={emergency}
               image={image}
               languages={languages}
+              gender={gender}
```

Comment:
Actually, I wasn´t sure if it would filter properly if the gender was not displayed in the card,. But if it works fine without being diplayed, It can be removed.